src/frontend/screens/Exchange/index.tsx

```diff
@@ -26,18 +26,15 @@ const m = defineMessages({
 export const SyncScreen = ({navigation}: NativeRootNavigationProps<'Sync'>) => {
   const wifiStatus = useLocalDiscoveryState(state => state.wifiStatus);
 
-  const {data: remoteArchive, isRefetching: remoteArchiveLoading} =
-    useGetRemoteArchives();
-
-  const hasRemoteArchive = remoteArchive && remoteArchive.length > 0;
-
   const hasInternetAccess = useNetInfo().isConnected;
 
   const {projectId} = useActiveProject();
   const syncState = useSyncState({projectId});
   const projectSettingsQuery = useProjectSettings();
 
-  if (!syncState || remoteArchiveLoading) {
```

Comment:
just noting the change here. i don't think this should have a major impact on this screen, but could be wrong!